### DISCUSSION TITLE
Safari iOS PWA Home Button Overlap

### DISCUSSION THREAD

**Discussion by parthpat12** [danny-avila/LibreChat#4897] [Troubleshooting]
### What happened?

When using LibreChat as PWA on iOS, text on the bottom is getting covered by home pill. Can we format this so it doesn’t overlap? 

### Steps to Reproduce

1) Visit LibreChat instance in iOS Safari
2) Add to Home Screen to store as PWA
![Screenshot 2024-12-06 at 8 25 43 PM](https://github.com/user-attachments/assets/50d925c7-41ac-498e-94fe-2fe9c0a595a1)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There isn't much real estate here for the mobile UI to be sacrificing even more of it. When viewed on a regular browser, this isn't an issue, and I can see this being the same for ChatGPT

**Comment by parthpat12** [comment]
After removing Privacy Policy and ToS block in config.yaml, it’s a little better. Is it possible to add same for version? Or is branding/version required? 
![Screenshot 2024-12-07 at 3 03 05 PM jpeg](https://github.com/user-attachments/assets/56b77b11-2d31-437a-b15f-4debf1dfe781)
